### PR TITLE
Update payment_terminal.json

### PR DESCRIPTION
**Pull request by evgenykatyshev**
Add PayBox for Georgia

### CODE REVIEW

File: data/brands/amenity/payment_terminal.json [hunk 1]
```diff
@@ -618,6 +618,17 @@
         "payment:cash:USD": "yes",
         "payment:coins": "no"
       }
+    },
+    {
+      "displayName": "ფეიბოქსი",
+      "locationSet": {"include": ["ge"]},
+      "tags": {
+        "amenity": "payment_terminal",
+        "brand": "ფეიბოქსი",
```
Comment (AntiCompositeNumber) [inline data/brands/amenity/payment_terminal.json:627]: Please add a brand:wikidata tag, creating a new Wikidata item if necessary. For information on editing Wikidata, please see https://github.com/osmlab/name-suggestion-index/wiki/Editing-Wikidata

**Review by AntiCompositeNumber — CHANGES_REQUESTED** [review]

File: data/brands/amenity/payment_terminal.json (hunk 1 above)
Comment (danieldegroot2) [reply data/brands/amenity/payment_terminal.json:627]: Would be good to do the same for
https://github.com/osmlab/name-suggestion-index/blob/0bfac56dbdad1aeae8bf1ce857e99869f734c8d4/data/brands/amenity/payment_terminal.json#L583
Oppa operates PayBox, Teso, Oppa Service, Asterbit.

Comment (evgenykatyshev) [reply data/brands/amenity/payment_terminal.json]: > Please add a brand:wikidata tag, creating a new Wikidata item if necessary. For information on editing Wikidata, please see https://github.com/osmlab/name-suggestion-index/wiki/Editing-Wikidata

Added Wikidata and operator

**Comment by Snowysauce** [comment]
As of this writing, there are two operator tags in the code and the Wikidata page for the brand was deleted.

**Comment by Snowysauce** [comment]
Closing as an abandoned PR based around an entry whose Wikidata item was deleted.

**Pull request closed without merging**